pkg/kubelet/cm/helpers.go

```diff
@@ -65,6 +65,7 @@ func getLibcontainerCgroupManager(cgroupConfig *CgroupConfig) (*cgroupfs.Manager
 	// Extract the cgroup resource parameters
 	resourceConfig := cgroupConfig.ResourceParameters
 	resources := &libcontainerconfigs.Resources{}
+	resources.AllowAllDevices = true
```

Comment:
nit: Move this line into your second commit - libcontainer cgroup manager implementation.